test/PowerShellEditorServices.Test.E2E/LanguageServerProtocolMessageTests.cs

```diff
@@ -51,15 +51,14 @@ public LanguageServerProtocolMessageTests(ITestOutputHelper output, LSPTestsFixt
             Diagnostics.Clear();
             TelemetryEvents = data.TelemetryEvents;
             TelemetryEvents.Clear();
-            _fixture = data;
-
             PwshExe = PsesStdioProcess.PwshExe;
         }
 
         public void Dispose()
         {
             Diagnostics.Clear();
             TelemetryEvents.Clear();
+            GC.SuppressFinalize(this);
```

Comment:
Should this ever run on a finalizer thread?  Is there a class destructor to finalize?  Just wondering if this is needed.

Reply:
I had a long conversation with Dongbo to learn all about this and while this doesn't cause a bug, the compiler barfs a warning about it, and this is the "correct" fix. If this were product code it would be more important, but I'm working to just get us to a zero-warning codebase. Slowly, but surely.